feat(phone): Add starter kit for the phone channel. (#166)
watson-developer-cloud/assistant-toolkit: integrations/phone/examples/README.md

## Examples

This folder contains a set of examples that are intended to demonstrate how to use various features of the Watson Assistant phone channel. Below is a list of these examples, along with a brief explanation of each and the different features that each example demonstrates.

### [Common Interactions](./common-interactions/)

> This example contains Watson Assistant phone integration actions demonstrate:

Author: ```suggestion
 This example contains Watson Assistant phone integration actions that demonstrate:
```

>  This example contains Watson Assistant phone integration actions that demonstrate:

Author: ```suggestion
This example contains Watson Assistant phone integration actions that demonstrate:
```

> This example contains Watson Assistant phone integration actions that demonstrate how to:

Reviewer: cut 'how to'

> This example contains Watson Assistant phone integration actions that demonstrate how to:

Reviewer: Watson Assistant phone integration actions demonstrate: 

(rest of sentence can be cut because you already said above that these are examples, plus each bullet point says 'how to' so 'how to' here isnt needed)

Author: ```suggestion
This example contains Watson Assistant phone integration actions demonstrate:
```

> This example contains Watson Assistant phone integration actions that demonstrate how to:

Author: Thanks Kat, I updated this in: https://github.com/watson-developer-cloud/assistant-toolkit/pull/166#discussion_r1285956115

> - How to use the phone and SMS channel for 2-factor authentication

Reviewer: two-factor, not 2-factor

Author: ```suggestion
- How to use the phone and SMS channel for two-factor authentication
```

- How to tune the Watson Speech-To-Text parameters in the Greetings action for background noise (`background_audio_suppression`) and increase the speech capture time (`end_of_phrase_silence_time`).

> - How to disable and enable Text-to-Speech barge-in

Author: ```suggestion
- How to disable and enable Text-to-Speech barge-in.
```

> - How to handle the input timeout from the phone channel via the `vgwPostResponseTimeout` keyword

Author: ```suggestion
- How to handle the input timeout from the phone channel via the `vgwPostResponseTimeout` keyword.
```

> - How to disconnect the call when the caller is done.

Reviewer: should be a period after every bullet.

authentication.
barge-in.
keyword.

### [Amazon Connect](./amazon-connect/)

> These instructions provide an example of how to build a custom contact control panel (CCP) that grabs the data from an Amazon Connect flow and uses it to display the conversation history with Watson Assistant. It can be used as a reference for more complex and customer-specific use cases. 

Reviewer: This custom contact control panel (CCP) grabs data from... Watson Assistant, and can be a reference for more complex and customer-specific cases.

Author: ```suggestion
This custom contact control panel (CCP) grabs data from an Amazon Connect flow and uses it to display the conversation history with Watson Assistant, and can be a reference for more complex and customer-specific cases.
```

> The example used in these instructions is based on this [blog](https://aws.amazon.com/blogs/contact-center/perform-an-external-screen-pop-with-amazon-connect/). 

Reviewer: cut 'used'

Author: ```suggestion
The example in these instructions is based on this [blog](https://aws.amazon.com/blogs/contact-center/perform-an-external-screen-pop-with-amazon-connect/). 
```
